### PR TITLE
Add email alert buttons

### PR DESCRIPTION
**Pull request by mobaig**
pupil_wellbeing__behaviour_and_attendance_-_gov_uk

## Hiding signup link on General information and guidance

education_and_funding_for_high_needs__special_educational_needs_and_disability__send__-_gov_uk

https://trello.com/c/YImQlqXa/152-1-add-the-email-link-to-navigation-pages
https://trello.com/c/lPlSbZnX/166-3-build-3-types-of-sign-up-pages-backend

### CODE REVIEW

File: app/controllers/taxons_controller.rb [hunk 1]
```diff
@@ -31,6 +31,7 @@ def taxon_overview_and_child_taxons(taxon)
           'title' => current_taxon_title,
           'description' => '',
           'base_path' => current_taxon_title.downcase.tr(' ', '-'),
+          'is_current_taxon' => true,
```
Comment (carvil) [inline app/controllers/taxons_controller.rb]: I would probably rename this to `has_tagged_content`, which is the trigger for having General Information and Guidance.

Comment (mobaig) [inline app/controllers/taxons_controller.rb]: renamed

**Comment by mobaig** [comment]
thanks @carvil and @dwhenry

**Comment by mobaig** [comment]
@carvil https://trello.com/c/wucG3iIm/180-visual-fixes

**Pull request merged**